src/Passes/qir/stdlib/lib/stdlib.ll

```diff
@@ -0,0 +1,507 @@
+; ModuleID = 'src/stdlib.c'
+source_filename = "src/stdlib.c"
+target datalayout = "e-m:o-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-apple-macosx11.0.0"
```

Comment:
My expectation is that nothing depending on this target triple for the code in question. Would it make sense to remove it and let llvm default to the current machine as needed?

Reply:
It is auto-generated when running the `clang`. The only way to remove it consistently is post-processing. We probably should decide on a triplet for quantum machines as it _does_ impact how optimisations are performed by LLVM.